### PR TITLE
Fix issue#4, add contribute.json to project

### PR DESCRIPTION
**Pull request by schalkneethling**
@alexgibson r?

### CODE REVIEW

File: contribute.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        "url": "https://github.com/mozilla/mozilla-infobar",
+        "license": "MPL2"
+    },
+    "participate": {
+        "docs": "https://github.com/mozilla/mozilla-infobar/blob/master/README.md",
+        "irc": "irc://irc.mozilla.org/#www",
+        "irc-contacts": [
+            "agibson",
+            "espressive",
+            "jpetto"
```
Comment (alexgibson) [inline contribute.json]: Suggest adding kohei too, since he wrote much of the original code here?

**Comment by alexgibson** [comment]
r+wc :mount_fuji:

File: contribute.json (hunk 1 above)
Comment (schalkneethling) [inline contribute.json]: Oh indeed! I completely forgot.

**Comment by schalkneethling** [comment]
Thx @alexgibson - updated

**Pull request merged**